btr-web/btr-common-components/components/bcros/inputs/Address/Index.vue

```diff
@@ -129,5 +129,33 @@ const regions = computed(() => {
 
 const addrAutoCompleted = (selectedAddr: BtrAddressI) => {
   Object.assign(address.value, selectedAddr)
+
+  formBus?.emit({ type: 'blur', path: props.name + '.country' })
+  formBus?.emit({ type: 'change', path: props.name + '.country' })
+
+  setTimeout(() => {
+    formBus?.emit({ type: 'blur', path: props.name + '.line1' })
+    formBus?.emit({ type: 'change', path: props.name + '.line1' })
+  }, 10)
+
+  setTimeout(() => {
+    formBus?.emit({ type: 'blur', path: props.name + '.line2' })
+    formBus?.emit({ type: 'change', path: props.name + '.line2' })
+  }, 20)
+
+  setTimeout(() => {
+    formBus?.emit({ type: 'blur', path: props.name + '.city' })
+    formBus?.emit({ type: 'change', path: props.name + '.city' })
+  }, 30)
+
+  setTimeout(() => {
+    formBus?.emit({ type: 'blur', path: props.name + '.postalCode' })
+    formBus?.emit({ type: 'change', path: props.name + '.postalCode' })
+  }, 40)
+
+  setTimeout(() => {
+    formBus?.emit({ type: 'blur', path: props.name + '.region' })
+    formBus?.emit({ type: 'change', path: props.name + '.region' })
+  }, 50)
```

Comment:
haha you don't like loops do you? :P

Reply:
haha, yeah, was thinking of putting this in the loop though. But got some issues when I tried and I wanted to make this one go away fast.
Its unfortunate I had to make them emit change values in different intervals otherwise only the last one would be triggered.